src/components/Article.jsx

```diff
@@ -38,6 +43,8 @@ const Article = ({user}) => {
 
     if(loading){
         return (<h1>LOADING</h1>)
+    } else if(error === 404){
```

Comment:
The error might not be a 404 if we make a bad request with requesting an id of a string, we get a 400 which is causing this error component to not work correctly. A better way would be to just say if(error)